docs/tutorials/intro_tutorial.rst

```diff
@@ -169,148 +206,230 @@ The beginning of both classes looks like this:
+
+Note: If the script file track has been utilized. The script will be
+utilized in within the python interactive session in the same directory
+as your source code file, and import the ``moneyModel`` class. Then
+create a model object and run it for a few steps.
 
 .. code:: python
 
    from money_model import MoneyModel
 
-Then create the model object, and run it for one step:
+   starter_model = MoneyModel(10)
+   starter_model.step()
+
+Create the model object, and run it for one step:
 
 .. code:: ipython3
 
-    empty_model = MoneyModel(10)
-    empty_model.step()
+    starter_model = MoneyModel(10)
+    starter_model.step()
 
 
 .. parsed-literal::
 
-    Hi, I am agent 5.
-    Hi, I am agent 2.
-    Hi, I am agent 4.
-    Hi, I am agent 8.
-    Hi, I am agent 0.
-    Hi, I am agent 1.
-    Hi, I am agent 7.
-    Hi, I am agent 9.
-    Hi, I am agent 3.
-    Hi, I am agent 6.
+    Hi, I am an agent, you can call me 9.
+    Hi, I am an agent, you can call me 2.
+    Hi, I am an agent, you can call me 4.
+    Hi, I am an agent, you can call me 7.
+    Hi, I am an agent, you can call me 3.
+    Hi, I am an agent, you can call me 8.
+    Hi, I am an agent, you can call me 5.
+    Hi, I am an agent, you can call me 6.
+    Hi, I am an agent, you can call me 0.
+    Hi, I am an agent, you can call me 1.
+
+
+.. code:: ipython3
+
+    # Run this step overnight and see what happens! Notice the order of the agents changes each time.
+    starter_model.step()
+
+
+.. parsed-literal::
+
+    Hi, I am an agent, you can call me 9.
+    Hi, I am an agent, you can call me 6.
+    Hi, I am an agent, you can call me 4.
+    Hi, I am an agent, you can call me 5.
+    Hi, I am an agent, you can call me 2.
+    Hi, I am an agent, you can call me 8.
+    Hi, I am an agent, you can call me 0.
+    Hi, I am an agent, you can call me 1.
+    Hi, I am an agent, you can call me 7.
+    Hi, I am an agent, you can call me 3.
 
 
 Exercise
 ^^^^^^^^
 
-Try modifying the code above to have every agent print out its
-``wealth`` when it is activated. Run a few steps of the model to see how
-the agent activation order is shuffled each step.
+Modifying the code below to have every agent print out its ``wealth``
+when it is activated.
+
+.. code:: ipython3
+
+    class MoneyAgent(mesa.Agent):
+        """An agent with fixed initial wealth."""
+
+        def __init__(self, unique_id, model):
+            # Pass the parameters to the parent class.
+            super().__init__(unique_id, model)
+
+            # Create the agent's variable and set the initial values.
+            self.wealth = 1
+
+        def step(self):
+            # The agent's step will go here.
+            # FIXME: Need to print the agent's wealth
+            print(f"Hi, I am an agent and I am broke!")
+
+Create a model for 12 Agents, and run it for a few steps to see the
+output.
+
+.. code:: ipython3
+
+    # Fixme: Create the model object, and run it
+
```

Comment:
Awesome, this dramatically reduces the mental barrier to making the tutorial active learning